apps/documentation/src/components/Sidebar/sidebarElements.ts

```diff
@@ -106,6 +106,11 @@ export const sidebarElements: Array<SidebarElement> = [
     label: 'Routing',
     href: '#focus',
     children: [
+      {
```

Comment:
Could you please move this object to the end of the array so it is the one with the least priority within the `Routing` section?